client/app/app.module.ts

```diff
@@ -4,17 +4,15 @@ import { FormsModule, ReactiveFormsModule } from '@angular/forms';
 import { HttpModule } from '@angular/http';
 import { RouterModule } from '@angular/router';
 
+import { routes } from './config/routes';
 import { AppComponent } from './app.component';
 import { HomeComponent } from './home/home.component';
 import { AboutComponent } from './about/about.component';
 import { DataService } from './services/data.service';
 
 import { ToastComponent } from './shared/toast/toast.component';
 
-const routing = RouterModule.forRoot([
-    { path: '',      component: HomeComponent },
-    { path: 'about', component: AboutComponent }
-]);
+const routing = RouterModule.forRoot(routes,  { useHash: true });
```

Comment:
I was trying to implement a 'pageNotFound' component, which loads in angular client side whenever a wildcard url is given. But it seems the request goes to the server side and shows a cannot GET /**  error. 

Giving the useHash: true allows wildcard urls to be handled by the client side angularjs itself, So that it loads a pageNotFound component.